playground/backend/cmd/server/server.go

```diff
@@ -50,10 +50,16 @@ func runServer() error {
 	})
 
 	grpclog.SetLoggerV2(grpclog.NewLoggerV2(os.Stdout, os.Stdout, os.Stderr))
-	handler := Wrap(grpcServer, getGrpcWebOptions())
+
```

Comment:
asking for an extra change: In line 22 in this file we're importing the Go logging library. Should we be using our own logger?

Reply:
Done.